iceberg/iceberg-handler/src/test/java/org/apache/iceberg/mr/hive/TestHiveIcebergStorageHandlerNoScan.java

```diff
@@ -1576,6 +1576,20 @@ public void testAlterTableToIcebergAndMetadataLocation() throws IOException {
         });
   }
 
+  @Test
+  public void testCTLT() {
+    // Create a normal table and add some data
+    shell.executeStatement("CREATE TABLE source(a int)");
+    shell.executeStatement("insert into source values(1)");
+
+    shell.executeStatement(String.format("CREATE TABLE dest LIKE source STORED BY ICEBERG %s ",
+        testTables.locationForCreateTableSQL(TableIdentifier.of("default", "dest"))));
+
+    // Try a select query and check if the table is empty .
+    String result = shell.executeAndStringify("select a from " + TableIdentifier.of("default", "dest").name());
```

Comment:
Could you please validate that the table is iceberg? Also, I would be interested in checking some table properties.

Reply:
Added